movieapp/src/containers/Sort/__tests__/Sort.test.tsx

```diff
@@ -0,0 +1,54 @@
+import renderer from 'react-test-renderer';
+import { describe } from '@jest/globals';
+import { Sort } from '../Sort';
+import { fireEvent, render } from '@testing-library/react';
+import fetchMock from 'jest-fetch-mock';
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => {
+    return {
+      t: (str: any) => str,
+      i18n: {
+        changeLanguage: () => new Promise(() => {})
+      }
+    };
+  }
+}));
+
+const mockDispatch = jest.fn();
+jest.mock('react-redux', () => ({
+  useSelector: jest.fn(),
+  useDispatch: () => mockDispatch
+}));
+
+jest.mock('react-router', () => ({
+  useNavigate: () => mockDispatch,
+  navigate: () => mockDispatch
```

Comment:
it's really strange that u use same mock reference to dispatch for every function